akka-distributed-data/src/multi-jvm/scala/akka/cluster/ddata/DurableDataSpec.scala

```diff
@@ -82,7 +82,7 @@ abstract class DurableDataSpec(multiNodeConfig: DurableDataSpecConfig)
   implicit val cluster = Cluster(system)
 
   val timeout = 5.seconds.dilated
-  val writeTwo = WriteTo(2, timeout)
+  val writeTwo = WriteTo(14, timeout) // initialization of lmdb can be very slow in CI environment
```

Comment:
AFAICS this increases the number of replica's written to, not the timeout - is that what you intended?